fix(2.15.1): improve sidebar_image display

virgilwashere opened the pull request:
There is a difference between the `2.15.0` and `2.15.1` templates that is affecting the displayed width of the sidebar_image.

This updates the 2.15.1 templatge to the previous behaviour in 2.15.0.

Closes #65

khaoua commented:
I agree, having the same issue on my side, is it working better now on your side?

virgilwashere commented:
@khaoua see #65 .  It's doing what I need.

khaoua commented:
Ok, great, the fix is available then ? should I download again whitelabeler to have the correction applicable ? or did you make the change directly in your file ?

virgilwashere commented:
@khaoua, 

Until the PR is accepted and merged, you'll need to make the changes locally.
However, you can do this:

```bash
# change to top level 'mautic-whitelabeler' directory
cd /var/www/mautic-whitelabeler
git apply -v --ignore-whitespace --inaccurate-eof --recount \
  https://github.com/nickian/mautic-whitelabeler/pull/66.diff
```

```console
$ git checkout master
Switched to branch 'master'
Your branch is up to date with 'upstream/master'.
$ git apply -v --ignore-whitespace --inaccurate-eof --recount \
>   https://github.com/nickian/mautic-whitelabeler/pull/66.diff
Checking patch templates/2.15.1/app/bundles/CoreBundle/Views/LeftPanel/index.html.php...
Applied patch templates/2.15.1/app/bundles/CoreBundle/Views/LeftPanel/index.html.php cleanly.
$ git diff
───────────────────────────────────────────────
modified: templates/2.15.1/app/bundles/CoreBundle/Views/LeftPanel/index.html.php
───────────────────────────────────────────────
@ templates/2.15.1/app/bundles/CoreBundle/Views/LeftPanel/index.html.php:16 @ $extraMenu = $view['menu']->render('extra');
 <!-- start: sidebar-header -->
 <div class="sidebar-header">
     <!-- brand -->
-    <a class="mautic-brand<?php echo (!empty($extraMenu)) ? ' pull-left pl-0 pr-0' : ''; ?>" href="#">
+    <a class="mautic-brand<?php echo (!empty($extraMenu)) ? ' pull-left pl-0 pr-0' : ''; ?>" href="#" style="text-align:center; padding:0;">
        <img src="{{sidebar_image}}" style="width:100%; max-width:{{sidebar_width}}px; margin:{{margin_top}}px {{margin_right}}px 0 {{margin_left}}px;" />
     </a>
```

If you want to disable scaling image, comments in #65 

Virgil

khaoua commented:
oh, ok, I see, nothing urgent on my side, so I hope it will get merged and considered for next mautic release, thanks a lot for the tips ;-)
